baseline/tf/classify/training/utils.py

```diff
@@ -101,16 +101,14 @@ def __init__(self, model_params, **kwargs):
 
         """
         super().__init__()
-
         if type(model_params) is dict:
             self.model = create_model_for('classify', **model_params)
         else:
             self.model = model_params
-
         self.sess = self.model.sess
         self.loss = self.model.create_loss()
         self.test_loss = self.model.create_test_loss()
-        self.global_step, train_op = optimizer(self.loss, colocate_gradients_with_ops=True, **kwargs)
+        self.global_step, train_op = optimizer(self.loss, colocate_gradients_with_ops=True, variables=self.model.trainable_variables, **kwargs)
```

Comment:
should this idiom be applied more broadly?  other tasks?

Reply:
I think it should be done but I would rather get this in and track that in another merge because there is probably a lot of testing that needs to be done for that https://github.com/dpressel/mead-baseline/issues/493